expected/cnpj.out

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE empresa(cnpj cnpj);
+\d empresa
+             Table "public.empresa"
+ Column | Type | Collation | Nullable | Default 
+--------+------+-----------+----------+---------
+ cnpj   | cnpj |           |          | 
+
+INSERT INTO empresa VALUES (cnpj '00000000000000');
+SELECT * FROM empresa;
+        cnpj        
+--------------------
+ 00.000.000/0000-00
+(1 row)
```

Comment:
Is this a valid CNPJ?? I guess we should check for all digits equal and forbid it, no??

Reply:
I misread the wikipedia page: https://pt.wikipedia.org/wiki/Cadastro_Nacional_da_Pessoa_Jur%C3%ADdica (Apesar de ser algo contra-intuitivo, existe um CNPJ com raiz 00.000.000 (ZERO), sendo que o mesmo é pertencente ao Banco do Brasil.), so it's not all zeroes, is just the number. I'll change the tests

About equal digits, I couldn't find any reference regarding this restriction.